### PR TITLE
Subscription #39

### PR DESCRIPTION
**Pull request by cbergmiller**
Subscription.unsubscribe() now supports a list of handles (I added two tests for that).
Docstrings in Subscription were updated and typings for handles were added.

### CODE REVIEW

File: asyncua/common/subscription.py [hunk 1]
```diff
@@ -135,7 +138,8 @@ def _call_datachange(self, datachange: ua.DataChangeNotification):
                     self._handler.datachange_notification(data.node, item.Value.Value.Value, event_data)
                 except Exception:
                     self.logger.exception("Exception calling data change handler")
-            elif hasattr(self._handler, "data_change"):  # deprecated API
+            elif hasattr(self._handler, "data_change"):
+                # deprecated API
                 self.logger.warning("data_change method is deprecated, use datachange_notification")
```
Comment (oroulet) [inline asyncua/common/subscription.py]: @cbergmiller we should remove these deprecated APIs in opcua-asyncio....

Comment (cbergmiller) [reply asyncua/common/subscription.py]: I will look into that!

**Comment by oroulet** [comment]
the tests passed on this PR, but these changes seems to break master: https://travis-ci.org/FreeOpcUa/opcua-asyncio

**Comment by oroulet** [comment]
@cbergmiller

**Pull request merged**